### PR TITLE
[hold for fe] Drush 11 Incompatible with D8

### PR DESCRIPTION
**Pull request by wordsmither**
Closes #

## Summary

**[Create Site](https://pantheon.io/docs/guides/drupal-9-hosted/create-site)** - Updated code sample to include drush

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Review by whitneymeredith — APPROVED** [review]
LGTM

File: source/partials/drupal/core-version-remain-on-d8.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ reviewed: ""
 
   ```bash{promptUser:user}
   composer require --no-update drupal/core-recommended:^9
+  composer require --no-update drush/drush:^10
```
Comment (EdwardAngert) [inline source/partials/drupal/core-version-remain-on-d8.md:13]: @wordsmither should this PR be close in favor of #8140 ?

**Review by EdwardAngert — CHANGES_REQUESTED** [review]
consider closing in favor of https://github.com/pantheon-systems/documentation/pull/8140/files#diff-5de48a9c5daeb5cc7677c2d93a0f4500abf0bc6cedd153c5d2291f5e46cc96e2R13

**Comment by wordsmither** [comment]
Not sure what happened here, but yes, this should be closed in favor of 8140

**Pull request closed without merging**